### PR TITLE
ST-568 Fix setting FEES in script

### PR DESCRIPTION
**Pull request by KitHat**

### CODE REVIEW

**Comment by ashcherbakov** [comment]
(ci) test this please

File: scripts/performance/perf_load/perf_client_fees.py [hunk 1]
```diff
@@ -45,7 +47,9 @@ def __init__(self, name, pipe_conn, batch_size, batch_rate, req_kind, buff_req,
         self._plugin_lib = kwargs.get("plugin_lib", "")
         self._plugin_init = kwargs.get("plugin_init", "")
         self._req_num_of_trustees = kwargs.get("trustees_num", 4)
-        self._set_fees = kwargs.get("set_fees", {})
+        fees = kwargs.get("set_fees", {})
+        self._auth_rule_metadata = {k: {"fees": random_string()} for k, _ in fees.items()}
+        self._set_fees = {self._auth_rule_metadata.get(k, {}).get('fees', None): v for k, v in fees.items()}
```
Comment (andkononykhin) [inline scripts/performance/perf_load/perf_client_fees.py]: I think it makes sense to assume that `[k]['fees']` is inside the dict or use one common loop to set both `self._auth_rule_metadata` and `self._set_fees` at once.

File: scripts/performance/perf_load/perf_client_fees.py [hunk 2]
```diff
@@ -200,7 +204,10 @@ def _restore_fees_from_req(self, req):
         get_fees_req = await payment.build_get_txn_fees_req(self._wallet_handle, self._test_did, self._payment_method)
         get_fees_resp = await ledger.sign_and_submit_request(self._pool_handle, self._wallet_handle, self._test_did,
                                                              get_fees_req)
-        self._pool_fees = json.loads(await payment.parse_get_txn_fees_response(self._payment_method, get_fees_resp))
+
+        type_alias_mapping = {v['fees']: k for k, v in self._auth_rule_metadata.items()}
```
Comment (andkononykhin) [inline scripts/performance/perf_load/perf_client_fees.py]: seems `alias_type_mapping` is more proper name here

File: scripts/performance/perf_load/perf_client_fees.py [hunk 3]
```diff
@@ -228,3 +236,9 @@ def _on_pool_create_ext_params(self):
                        "pool_fees": self._pool_fees})
         self._logger.info("_on_pool_create_ext_params done {}".format(params))
         return params
+
+
+def random_string(string_length=10):
```
Comment (andkononykhin) [inline scripts/performance/perf_load/perf_client_fees.py]: I think it's better to re-use/extend helper API from `scripts/performance/perf_load/perf_utils.py`: https://github.com/hyperledger/indy-node/blob/f08c6cf2a64d290bf015dfbb9fd9222da5bdfb1f/scripts/performance/perf_load/perf_utils.py#L54

**Pull request merged**